scripts/s3BucketOptions/launch.bash

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+#
+
+set -o nounset
+set -o pipefail
+IFS=$'\n\t'
+
+if [ -f ".venv/bin/activate" ] ; then
+	sleep 0
```

Comment:
Q: why the sleep is necessary here ? why not just check that the file does not exists?

Reply:
It is only for readability, since if/else checks in bash are a bit tricky. `sleep 0` is an empty statement, it does nothing
This reads as:
`If the venv exists, do nothing, else, do this/that`